mksfosinitrd.sh

```diff
@@ -84,6 +85,8 @@ cd "$TMP_DIR"
 # Copy local files to be added to initrd. If you add more, add also to TOOL_LIST.
 cp -a "$OLD_DIR"/sbin .
 cp -a "$OLD_DIR"/res .
+mkdir etc
```

Comment:
As you may see above:
rm -rf "$TMP_DIR"
mkdir "$TMP_DIR"
cd "$TMP_DIR"

So we are in the empty directory.